docs/custom.css

```
@@ -1949,3 +1953,88 @@ a.learn-sidebar-item.is-active {
+
+.mastra-search-button svg {
+  width: 16px;
+  height: 16px;
+}
+
+/* Collapse to an icon-only trigger below the large breakpoint. */
+.mastra-search-button > span {
+  display: none;
+}
+
+@media (min-width: 62.25rem) {
```

Stylelint (17.14.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)